dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2019 Philippe Retornaz <philippe@shapescale.com>
+ *
+ * Based on stm32l0.dtsi:
+ * Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
```

Comment:
This is nice but please remove.
Reason is all these files are copy/pasted from each others. If we want to keep track of every contributor we'll have a long list of people that only copy/pasted the previous folks.
So, for files created, specially in case of porting new stuff, we prefer to keep the copyright of the one doing the effort of porting and testing. Apply everywhere.